src/CoreBundle/private/library/classes/TCMSListManager/TCMSListManagerFullGroupTable.class.php

```diff
@@ -48,39 +48,29 @@ public function Init(&$oTableConf)
         $_SESSION['_tmpCurrentTableName'] = $this->oTableConf->sqlData['name']; // needed for the callback functions...
         $_SESSION['_tmpCurrentTableID'] = $this->oTableConf->sqlData['id']; // needed for the callback functions...
 
-        $isTableCacheChangeRequest = $this->isTableCacheChangeRequest();
-
-        $oOldTableObj = null;
-        $sListCacheKey = $this->GetListCacheKey();
-
-        if ($this->bListCacheEnabled && CMS_ACTIVE_BACKEND_LIST_CACHE) {
-            if (!array_key_exists('_listObjCache', $_SESSION)) {
-                $_SESSION['_listObjCache'] = array();
-            }
-            $objectInSession = (array_key_exists($sListCacheKey, $_SESSION['_listObjCache']));
-
-            if ($objectInSession) {
-                $tmp = base64_decode($_SESSION['_listObjCache'][$sListCacheKey]);
-                $oOldTableObj = unserialize(gzuncompress($tmp));
-            }
+        $listCacheKey = $this->GetListCacheKey();
+        $cachedTableObj = null;
+        if (true === $this->isTableCachingEnabled()) {
+            $cachedTableObj = $this->getTableFromSessionCache($listCacheKey);
         }
 
         $aOrderData = array();
-        if (!is_null($oOldTableObj) && false !== $oOldTableObj) {
-            $aOrderData = $oOldTableObj->orderList;
+        if (null !== $cachedTableObj) {
+            $aOrderData = $cachedTableObj->orderList;
         }
 
         // table is not in cache, load it
-        if (is_null($oOldTableObj) || false === $oOldTableObj) {
+        if (null === $cachedTableObj) {
             $this->CreateTableObj();
+            $this->PostCreateTableObjectHook();
```

Comment:
CreateTableObj is a public method - so anyone could be calling it. By removing the PostCreateTableObjectHook you have introduced an unexpected change. Although I agree, that the call to the post create would be better placed here, I don't think we should change it in a patch level version. In any case, it would have to be documented in the upgrade documentation

Reply:
ok, changed back